apps/extension/src/entrypoints/options/pages/translation/personalized-prompt.tsx

```diff
@@ -52,20 +56,41 @@ function PromptList() {
   const translateConfig = useAtomValue(configFields.translate)
   const promptsConfig = translateConfig.promptsConfig
   const patterns = promptsConfig.patterns
+  const [selectedPrompts, setSelectedPrompts] = useState<string[]>([])
 
   return (
     <section className="w-full">
-      <header className="w-full text-end mb-4">
+      <header className="w-full text-end mb-4 gap-3 flex justify-end">
+        <ImportPrompts />
+        <ExportPrompts selectedPrompts={selectedPrompts} />
         <ConfigurePrompt />
       </header>
       <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4 max-h-96 overflow-auto p-2">
         {
           patterns.map(pattern => (
             <Card className="h-fit gap-4" key={pattern.id}>
               <CardHeader className="grid-rows-1">
-                <CardTitle className="truncate leading-relaxed">
+                <CardTitle className="truncate leading-relaxed gap-3 flex items-center">
                   {
-                    isDefaultPrompt(pattern.id) ? i18n.t('options.translation.personalizedPrompt.default') : pattern.name
+                    isDefaultPrompt(pattern.id)
+                      ? i18n.t('options.translation.personalizedPrompt.default')
+                      : (
+                          <>
```

Comment:
better to use `span` or `div` cause the component `Checkbox` and the text `pattern.name` are combination of an "check" action